README.md

```diff
@@ -185,6 +185,15 @@ Alternatively, to run a "smaller" set of tests, from the main repo folder run `p
 
 Note that some figures in the original paper were generated using the Python code at https://github.com/motiwari/BanditPAM-python. That code is not pretty, nor is it maintained. It only exists for reference and for reproducibility of the plots.
 
+## Documentation
+
+The documents of BanditPAM can be found here: 
+* Detailed description of BanditPAM: https://github.com/ThrunGroup/BanditPAM/blob/main/docs/long_desc.rst
```

Comment:
Instead of pointing to this, could you suggest that the user open up BanditPAM/docs/html/index.html when they've downloaded the repo locally? We have good docs generated via doxygen: 

![image](https://user-images.githubusercontent.com/13426340/123557544-6a498480-d75f-11eb-8e1c-bedf8ecac65a.png)